easybuild/easyconfigs/b/BinSanity/BinSanity-0.3.3-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,51 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'BinSanity'
+version = '0.3.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/edgraham/BinSanity/wiki'
+description = """BinSanity contains a suite a scripts designed to cluster contigs generated from
+metagenomic assembly into putative genomes."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/edgraham/BinSanity/archive/']
+sources = ['0d529465b5b1fa004e8fb465511041ca5098a764.tar.gz']
```

Comment:
please let EasyBuild rename this after download:

```python
sources = [{
    'download_filename': '0d529465b5b1fa004e8fb465511041ca5098a764.tar.gz',
    'filename': SOURCE_TAR_GZ,
}]
```

And maybe add a comment on why you're downloading via commit ID?